### ISSUE TITLE
Rate is output in sm3/s? 

### ISSUE THREAD

**Issue by goncalvesmachadoc** [OPM/opm-common#1334]
We were comparing the simulations results from April's release to the newest release. The injection rates output by the old version looks like what we would expect, with rates up to 50m3/day or so as described in the schedule input. The rates from the new version were much lower. We checked, they seem to be actually exported in m3/sec to the summary file.

**Comment by alfbr** [comment]
I have been comparing a number of simulations with master the last couple of weeks, and not noticed that difference. Are you able to narrow down what causes it, is it present on all models you test?

**Comment by bska** [comment]
> The injection rates output by the old version looks like what we would expect, with rates up to 50m3/day or so as described in the schedule input.

Are you simulating a lab setup?  I'm just asking because 50sm³/day is a fairly low rate for a field case.

**Comment by goncalvesmachadoc** [comment]
We are simulating the Egg model with metric system (those are common rates for this model). We will go ahead and test other models using metric system now.

**Comment by bska** [comment]
> We are simulating the Egg model with metric system (those are common rates for this model)

I see.

As a quick test, I created a synthetic model with rates in the same order of magnitude and ran it in current master&mdash;i.e., **not** the 2019.10 release.  Using the [`summary`](https://github.com/OPM/opm-common/blob/7a648cae7f75a067a3bff091d3db1101fc8b1fc0/test_util/summary.cpp) program to analyse the injection rates (water injectors constrained by surface water rate) I then got
```sh
$ summary EGG_STANDIN TIME WWIR:INJECT{1..4}
--          TIME    WWIR:INJECT1    WWIR:INJECT2    WWIR:INJECT3    WWIR:INJECT4
        1.000000       79.500000       79.500000       79.500000       79.500000
        4.000000       79.500000       79.500000       79.500000       79.500000
       13.000000       79.500000       79.500000       79.500000       79.500000
       30.000000       79.500000       79.500000       79.500000       79.500000
       60.000000       79.500000       79.500000       79.500000       79.500000
       90.000000       79.500000       79.500000       79.500000       79.500000
      120.000000       79.500000       79.500000       79.500000       79.500000
      150.000000       79.500000       79.500000       79.500000       79.500000
      180.000000       79.500000       79.500000       79.500000       79.500000
      210.000000       79.500000       79.500000       79.500000       79.500000
      240.000000       79.500000       79.500000       79.500000       79.500000
      270.000000       79.500000       79.500000       79.500000       79.500000
      300.000000       79.500000       79.500000       79.500000       79.500000
      330.000000       79.500000       79.500000       79.500000       79.500000
      360.000000       79.500000       79.500000       79.500000       79.500000
```
Of course, this could just be a systematic error in the I/O code that cancels out a unit conversion error, but it does at least suggest that we're not completely off.

**Comment by bska** [comment]
@goncalvesmachadoc : Are you still seeing the low rate symptoms?

**Comment by goncalvesmachadoc** [comment]
Yes, in selected cases the rates are outputted in m3/s. We are trying to figure out which feautre is causing it.

**Comment by bska** [comment]
>> Are you still seeing the low rate symptoms?
>
> Yes, in selected cases the rates are outputted in m3/s.

Very well, thanks for the update.  Do please keep me informed of any progress you make and fee free to send me an e-mail if you have further questions.  I'll assist in any way I can.
